Source/Initialization/WarpXInitData.cpp

```diff
@@ -246,7 +246,8 @@ WarpX::InitNCICorrector ()
 void
 WarpX::InitFilter (){
     if (WarpX::use_filter){
-        WarpX::bilinear_filter.npass_each_dir = WarpX::filter_npass_each_dir;
+        for (int i = 0; i < AMREX_SPACEDIM; ++i)
+            WarpX::bilinear_filter.npass_each_dir[i] = WarpX::filter_npass_each_dir[i];
```

Comment:
Oh, was that an out-of-bounds? The assignment does work not work anymore or did it throws a warning?

Safer in release mode if this compiles and behaves like `std::array`:
```suggestion
            WarpX::bilinear_filter.npass_each_dir.at(i) = WarpX::filter_npass_each_dir.at(i);
```